src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import classNames from 'classnames';
+import { switchError } from '../../utils/switchError';
+import { Error } from '../../types/Error';
+
+type Props = {
+  error: Error | null;
+};
+
+export const ErrorNotification: React.FC<Props> = ({ error }) => {
+  const errorType = !error
```

Comment:
The `switchError` function is being called regardless of error existence, which might be unnecessary and could lead to unexpected behavior. Consider refactoring this line to avoid calling the function when there's no error.